clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -3160,6 +3160,23 @@ static void handleWorkGroupSize(Sema &S, Decl *D, const ParsedAttr &AL) {
   S.addIntelTripleArgAttr<WorkGroupAttr>(D, AL, XDimExpr, YDimExpr, ZDimExpr);
 }
 
+// The lambda returns a DupArgResult value; Same means the args have the same
+// value, Different means the args do not have the same value, and Unknown
+// means that the args cannot (yet) be compared.
+enum class DupArgResult { Unknown, Same, Different };
```

Comment:
Should we put this definition to unnamed namespace to make it internal for this TU?

Reply:
I'm not opposed, but the usual advice (https://llvm.org/docs/CodingStandards.html#anonymous-namespaces) is to only do this for classes. I think using an anonymous namespace would add a bit of clutter though, so before I add it, I'm wondering if you think it's necessary or not.

Reply:
It doesn't seem to be necessary as `AreArgValuesIdentical` is already specified as `static`.